index.js

```diff
@@ -96,7 +95,6 @@ const Minesweeper = {
 
 	, submit: function () {
 		this.done = true
-		this.aborted = false
```

Comment:
The reason for this that the code did not depend on the default values of `this.done` and `this.aborted`. Please put it back in.